core/src/components/side-menu/side-menu.stories.tsx

```diff
@@ -131,7 +131,8 @@ const Template = ({ persistent, collapsible }) =>
 
       <i style="color:white">Header items omitted for brevity. See patterns/navigation</i>
 
-      <tds-header-brand-symbol slot="end" link-href="https://scania.com" aria-label="Scania - red gryphon on blue shield">
+      <tds-header-brand-symbol slot="end" aria-label="Scania - red gryphon on blue shield">
```

Comment:
The aria label should be moved to the `<a>` tag since that is the element with the image.